### DISCUSSION TITLE
Switching from ds to coqui guide

### DISCUSSION THREAD

**Discussion by betimd** [coqui-ai/STT#2031] [Q&A] [answered]
Hi there,

First of all I wish you all the best @ new venture in Coqui.

Is there any guide on how to switch from Deepspeech to Coqui? Or we can just do the normal training with new libs (instead of ds stt)?

In terms of pbmm model files, is there any tool to convert to tflite or we can use same file structure as we did in ds?

Thanks in advance!

**Comment by reuben** [comment, marked as answer]
You should be able to just switch to the new libs, eg `import stt` instead of `import deepspeech`, with no changes. For training, some small changes might be needed, as we switched from absl-py to coqpit for configuration and flag handling. The biggest change is that boolean flags must be specified explicitly, for example `--train_cudnn true` instead of just `--train_cudnn`. For TFLite, you can simply re-export old checkpoints to get an updated model in the correct format.
